src/transform/CollapseTable.css

```diff
@@ -179,7 +179,8 @@ div.pagelib_collapse_table_container {
    but only for tables we collapse. */
 .pagelib_collapse_table_container table {
     display: block;
-    width: 100% !important
+    width: 100% !important;
+    height: unset !important;
```

Comment:
I think Android had a compatibility issue with `unset` and `initial` was preferred... https://github.com/wikimedia/wikimedia-page-library/pull/176